x/profiles/types/models_params_test.go

```diff
@@ -10,170 +9,163 @@ import (
 	"github.com/desmos-labs/desmos/x/profiles/types"
 )
 
-func TestDefaultParams(t *testing.T) {
-	nameSurnameParams := types.NewNicknameParams(sdk.NewInt(2), sdk.NewInt(1000))
-	nicknameParams := types.NewDTagParams("^[A-Za-z0-9_]+$", sdk.NewInt(3), sdk.NewInt(30))
-	bioParams := sdk.NewInt(1000)
-
-	params := types.NewParams(nameSurnameParams, nicknameParams, bioParams)
-
-	require.Equal(t, params, types.DefaultParams())
-}
-
 func TestValidateParams(t *testing.T) {
```

Comment:
```suggestion
func TestParams_ValidateParams(t *testing.T) {
```

I think we should follow the same naming format.